mantis/cli/estimate_stabilization.py

```diff
@@ -0,0 +1,376 @@
+import itertools
+import multiprocessing as mp
+
+from functools import partial
+from pathlib import Path
+from typing import Tuple
+
+import click
+import numpy as np
+import pandas as pd
+
+from iohub.ngff import open_ome_zarr
+from pystackreg import StackReg
+from waveorder.focus import focus_from_transverse_band
+
+from mantis.analysis.AnalysisSettings import StabilizationSettings
+from mantis.cli.parsing import input_position_dirpaths, output_filepath
+from mantis.cli.utils import model_to_yaml
+
+NA_DET = 1.35
```

Comment:
These are hardcoded for now... they are used for the focus finder. 
Should we fix these now or let it be(?)

Some of these parameters would nice to have in the zarr (i.e magnification, pixel size, NA_Detection, illumination).

Reply:
#144 We will fix this in a later PR.